core/src/subgraph/inputs.rs

```diff
@@ -0,0 +1,22 @@
+use graph::components::store::WritableStore;
+use graph::{
+    blockchain::Blockchain,
+    components::store::{DeploymentLocator, SubgraphFork},
+    data::subgraph::{SubgraphFeature, UnifiedMappingApiVersion},
+    prelude::BlockNumber,
+};
+use std::collections::BTreeSet;
+use std::sync::Arc;
+
+pub struct IndexingInputs<C: Blockchain> {
```

Comment:
Are we planning on expanding this module further to include more stuff? Maybe it could be merged with the `context.rs` module introduced in this PR.

Reply:
Do you have any suggestion for the name of this module (`context` + `state` + `inputs`)?
Is your PR approved, what's the number/link of it?

Reply:
@maoueh I'll merge this then and change this on the next refactor PR, since I do intend on restructuring all of those anyway